### PR TITLE
Add fr.json

### PR DESCRIPTION
**Pull request by KikicheAlaFraise**

### CODE REVIEW

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    },
+    "NOTIFICATIONS": {
+      "MESSAGE_FROM": "Message de"
+    },
+    "PARTY_MENU": {
+      "ACCEPT_INVITE": "Accepter l'invitation de {{name}}",
+      "LAUNCH_PARTY": "Lancer la partie",
+      "INVITES": "Invités",
+      "NO_INVITES": "Vous n'avez pas d'invitations",
+      "EXPERIENCE_WARNING": "Pour une expérience de quête optimale, veillez à lire le guide de jeu sur notre wiki, que vous pouvez trouver via notre site web, <b>skyrim-together.com</b>. Voici un petit résumé des directives :",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:42]:
```diff
- "EXPERIENCE_WARNING": "Pour une expérience de quête optimale, veillez à lire le guide de jeu sur notre wiki, que vous pouvez trouver via notre site web, <b>skyrim-together.com</b>. Voici un petit résumé des directives :",
+ "EXPERIENCE_WARNING": "Pour une expérience de quête optimale, veuillez lire le guide de jeu sur notre wiki, que vous pouvez trouver via notre site web, <b>skyrim-together.com</b>. Voici un petit résumé des directives :",
```

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      "PUBLIC_SERVERS": "Serveurs publics",
+      "ERROR": {
+        "CONNECTION": "Impossible de se connecter au serveur spécifié. Veuillez vous assurer que vous avez entré l'adresse correcte et que vous ne rencontrez pas de problèmes de réseau.",
+        "VERSION_MISMATCH": "Vous ne pouvez pas vous connecter au serveur car il ne dispose pas de la même version de Skyrim Together que vous.",
+        "INVALID_ADDRESS": "Le format de l'adresse est invalide. (Adresse:Port)"
+      }
+    },
+    "DISCONNECT": {
+      "PROMPT": "Vous êtes sûr de vouloir vous déconnecter ?",
+      "PROCEED": "Allez-y",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:28]:
```diff
- "PROCEED": "Allez-y",
+ "PROCEED": "Quitter",
```

"Quitter" which means "Quit / Leave" is generally the term used to proceed to quit/leave a software in french.

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+      "ACCEPT_INVITE": "Accepter l'invitation de {{name}}",
+      "LAUNCH_PARTY": "Lancer la partie",
+      "INVITES": "Invités",
+      "NO_INVITES": "Vous n'avez pas d'invitations",
+      "EXPERIENCE_WARNING": "Pour une expérience de quête optimale, veillez à lire le guide de jeu sur notre wiki, que vous pouvez trouver via notre site web, <b>skyrim-together.com</b>. Voici un petit résumé des directives :",
+      "GUIDELINES": {
+        "PARTY": "En créant un groupe, vous devenez en quelque sorte l'hôte, autrement dit le chef de groupe.",
+        "PARTY_LEADER_TALK": "Seul le chef du groupe peut parler aux PNJ des quêtes pour progresser dans ces dernières.",
+        "PARTY_LEADER_LOOT_QUESTITEMS": "Seul le chef de groupe peut récupérer les objets de quête.",
+        "KEEP_PARTY_LEADER": "Tenez-vous-en à un seul chef de groupe par partie.",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:47]: Could i get the context for this?

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+      "MESSAGE_FROM": "Message de"
+    },
+    "PARTY_MENU": {
+      "ACCEPT_INVITE": "Accepter l'invitation de {{name}}",
+      "LAUNCH_PARTY": "Lancer la partie",
+      "INVITES": "Invités",
+      "NO_INVITES": "Vous n'avez pas d'invitations",
+      "EXPERIENCE_WARNING": "Pour une expérience de quête optimale, veillez à lire le guide de jeu sur notre wiki, que vous pouvez trouver via notre site web, <b>skyrim-together.com</b>. Voici un petit résumé des directives :",
+      "GUIDELINES": {
+        "PARTY": "En créant un groupe, vous devenez en quelque sorte l'hôte, autrement dit le chef de groupe.",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:44]: Could i get the context for this?

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+        "PARTY_LEADER_TALK": "Seul le chef du groupe peut parler aux PNJ des quêtes pour progresser dans ces dernières.",
+        "PARTY_LEADER_LOOT_QUESTITEMS": "Seul le chef de groupe peut récupérer les objets de quête.",
+        "KEEP_PARTY_LEADER": "Tenez-vous-en à un seul chef de groupe par partie.",
+        "ONE_PARTY_PER_SERVER": "Tenez-vous-en à une partie par serveur."
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:48]: Could i get the context for this?
From the key name, i understand that the context is a party, but partie != party in french, a party is a groupe while a partie is a game round

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+      "NO_PLAYERS_IN_PARTY": "Il n'y a encore personne dans votre groupe. Invitez-les via la liste des joueurs !",
+      "TABLE_HEADERS": {
+        "LEVEL": "Niv.",
+        "NAME": "Nom",
+        "LOCATION": "Localisation"
+      },
+      "ACTIONS": {
+        "KICK": "Exclure",
+        "TELEPORT": "Téléporter",
+        "MAKE_LEADER": "En faire le chef de groupe"
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:59]: Could also be
"Donner le role de chef" which means "Give leader role" not sure whats best without more context

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+      },
+      "LEAVE_PARTY": "Quitter le groupe"
+    },
+    "PARTY_LIST": {
+      "NO_OTHER_PLAYERS": "Il n'y a personne ici à part nous poulets ",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:64]: Not sure if its the joke is using "nous" as "us", to describe chickens or if its a misspelling, could probably do without the joke entirely tbh

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+        "NAME": "Nom du serveur",
+        "PLAYERS": "Joueurs / Max.",
+        "COUNTRY": "Pays",
+        "FAVORITE": "Favoris"
+      },
+      "JOIN": "Rejoindre"
+    },
+    "SETTINGS": {
+      "AUDIO": "Audio",
+      "MUTE_UI": "Interface utilisateur muette",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:104]: Reads weird, but cant think of something better to replace it with right now

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+      "UI_VOLUME": "Volume de l'interface utilisateur",
+      "PARTY": "Groupe",
+      "SHOW_PARTY": "Afficher le groupe",
+      "AUTO_HIDE_PARTY": "Masquer automatiquement le groupe",
+      "AUTO_HIDE_PARTY_LENGTH": "Après...",
+      "PARTY_ANCHOR": "Position de l'UI de groupe",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:110]:
```diff
- "PARTY_ANCHOR": "Position de l'UI de groupe",
+ "PARTY_ANCHOR": "Position de l'interface de groupe",
```

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+      "LANGUAGE": "Langue",
+      "DEBUG": "Debug",
+      "SHOW_DEBUG_INFO": "Afficher les informations de débogage",
+      "BACK": "Retour"
+    }
+  },
+  "SERVICE": {
+    "CLIENT": {
+      "CONNECTION_LOST": "Connexion perdue, tentative de reconnexion. {{remainingReconnectionAttempt}} tentatives.",
+      "CONNECTED": "Connexion réussie au serveur.",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:130]:
```diff
- "CONNECTED": "Connexion réussie au serveur.",
+ "CONNECTED": "Connexion au serveur réussite.",
```

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+      "DISCONNECTED": "Déconnectée du serveur."
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:131]:
```diff
- "DISCONNECTED": "Déconnectée du serveur."
+ "DISCONNECTED": "Déconnecté(e) du serveur."
```
french genders correctness :)

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+    },
+    "GROUP": {
+      "LEVEL_UP": "{{name}} a atteint le niveau {{level}}.",
+      "ALREADY_IN_GROUP": "Vous êtes déjà dans un groupe. Pour rejoindre un autre groupe, veuillez quitter votre groupe actuel.",
+      "KICK_NO_PARTY_LEADER": "YVous ne pouvez pas exclure les autres membres car vous n'êtes pas le chef du groupe.",
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:136]:
```diff
- "KICK_NO_PARTY_LEADER": "YVous ne pouvez pas exclure les autres membres car vous n'êtes pas le chef du groupe.",
+ "KICK_NO_PARTY_LEADER": "Vous ne pouvez pas exclure les autres membres car vous n'êtes pas le chef du groupe.",
```
Typo

File: Code/skyrim_ui/src/assets/i18n/fr.json [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+      "MAKE_LEADER_NO_PARTY_LEADER": "Vous ne pouvez pas faire d'un autre membre le chef, car vous n'êtes pas le chef... "
```
Comment (Banou26) [inline Code/skyrim_ui/src/assets/i18n/fr.json:137]:
```diff
- "MAKE_LEADER_NO_PARTY_LEADER": "Vous ne pouvez pas faire d'un autre membre le chef, car vous n'êtes pas le chef... "
+ "MAKE_LEADER_NO_PARTY_LEADER": "Vous ne pouvez pas faire d'un autre membre le chef du groupe, car vous n'êtes pas le chef"
```
Not sure why the 3dots were used here

**Review by Banou26 — COMMENTED** [review]
I'm really not the best at french even tho I'm french, so take everything i say with a grain of salt :)

File: Code/skyrim_ui/src/assets/i18n/fr.json (hunk 5 above)
Comment (KikicheAlaFraise) [reply Code/skyrim_ui/src/assets/i18n/fr.json:48]: Yep d'ailleurs, "Tenez-vous-en à un seul chef de groupe par partie." => "un seul chef de goupe par serveur", serait plus exacte car il est conseiller de faire une "partie"(Sauvegarde) par serveur.

"Tenez-vous-en à une partie par serveur." => "Tenez-vous en à une sauvegarde par serveur" également pour le coup, nop ? :)

J'utilise "partie" car pour moi, c'était "plus simple" à comprendre mais j'me trompe peut être ^^

File: Code/skyrim_ui/src/assets/i18n/fr.json (hunk 10 above)
Comment (such-dev) [reply Code/skyrim_ui/src/assets/i18n/fr.json:130]: "réussite" should be "réussie".

**Pull request merged**